etc/cherrytree.profile

```diff
@@ -0,0 +1,14 @@
+whitelist ${HOME}/cherrytree
+whitelist ${HOME}/.config/cherrytree/
+whitelist ${HOME}/.local/share/
+include /etc/firejail/disable-mgmt.inc
+include /etc/firejail/disable-secret.inc
+include /etc/firejail/disable-common.inc
+include /etc/firejail/disable-devel.inc
+caps.drop all
+seccomp
+protocol unix,inet,inet6,netlink
```

Comment:
Why does a note taking application need inet sockets?
Couldn't there a "net none" line be added?